systems/robotInterfaces/QPLocomotionPlan.m

```diff
@@ -44,47 +44,42 @@
       obj.robot = robot;
       S = load(obj.robot.fixed_point_file);
       obj.qtraj = S.xstar(1:obj.robot.getNumPositions());
-      obj.default_qp_input = atlasControllers.QPInputConstantHeight();
-      obj.default_qp_input.whole_body_data.q_des = zeros(obj.robot.getNumPositions(), 1);
+      obj.default_qp_input_ = atlasControllers.QPInputConstantHeight();
```

Comment:
`default_qp_input` is now a getter function (so that, someday, it can be implemented in c++), and the underlying variable it returns has the underscore